wordpress-mobile/WordPress-iOS
# Add support for actually purchasing the domain in the free credit redemption flow.

Goes in pair with https://github.com/wordpress-mobile/WordPressKit-iOS/pull/54.

The domain redemption flow actually purchases the domain now! It also changes the primary domain on a site to the newly purchased one.

It's a bit hard to verify in the app at the moment (the whole thing is very much still WIP!), but you can verify that the domain gets purchased by the email you get from WPCom billing and see that the primary domain gets changed in WPCom interface.

To test:
Go to `Plugins` on a WPCom Biz site that has a free domain credit to use.
Purchase a domain (preferably .blog, so it's cheaper for us, but also take other TLD's for a spin!)
Verify on WPCom that the domain was purchased

Optional: Cancel the domain in SA afterwards so we don't get billed for it ;P
